### ISSUE TITLE
When is 'ceil_mode' for AvgPool2D going to be supported ?  

### ISSUE THREAD

**Issue by kyuhyoung** [Unity-Technologies/barracuda-release#289]
Hello. I am trying to use a pre-trained pytorch model in barracuda.
Since the pre-trained model has been made with "AvgPool2D(...... ceil_mode = True)" and current barracuda does NOT support "ceil_mode", I am seeing errors saying "Off-axis dimension must match ......"
I would like to know
1. When "ceil_mode" is going to be supported ?
2. If I can modify the current barracuda so that I can make a custom barracuda with "ceil_mode".  (For me, adding "ceil_mode" dose NOT seem to be such a difficult thing.)
Thank you.

**Comment by Arminasss** [comment]
Hi @kyuhyoung! As of right now yes, we don't support ceiling mode for AvgPool2D operator. We are planning to implement support for ceiling mode for the AvgPool2D operator later in the future, but we can't provide an ETA for it.
As for your other question - yes, you can easily modify the Barracuda package since most of the codebase is written in C# language. You should be able to find the codebase in our package contents. Although it is worth mentioning that we are planning to enable users to implement their own operators and extra functionality through API in Barracuda 4.0 version. 
Hope I was able to answer your questions. Cheers!
